src/handlers/next/process-saleor-webhook.ts

```diff
@@ -0,0 +1,121 @@
+import { NextApiRequest } from "next";
+import getRawBody from "raw-body";
+
+import { APL } from "../../APL";
+import { AuthData } from "../../APL/apl";
+import { getSaleorHeaders } from "../../headers";
+import { verifySignature } from "../../verify-signature";
+import { getBaseURL } from "./get-base-url";
+
+// TODO: Resolve shadow lint issue
+// eslint-disable-next-line no-shadow
+export enum SaleorWebhookError {
+  MISSING_HOST_HEADER,
+  MISSING_DOMAIN_HEADER,
+  MISSING_EVENT_HEADER,
+  MISSING_PAYLOAD_HEADER,
+  MISSING_SIGNATURE_HEADER,
+  MISSING_REQUEST_BODY,
+  WRONG_EVENT,
+  NOT_REGISTERED,
+  SIGNATURE_VERIFICATION_FAILED,
+  WRONG_METHOD,
+}
```

Comment:
I'd say that we should entirely avoid enums, but if there is any actual reason - they should be named (`MISSING_HOST_HEADER="MISSING_HOST_HEADER"`)